src/ballet/toml/fd_toml.c

```diff
@@ -1282,14 +1293,18 @@ fd_float_parse_float_special( fd_toml_parser_t * parser,
   fd_toml_advance_inline( parser, 3UL );
 
   if( 0==memcmp( str, "inf", 3 ) ) {
-    FD_LOG_WARNING(( "float infinity is unsupported" ));
-    *pres = base * FLT_MAX;
+    if( FD_UNLIKELY( !fd_pod_insert_cstr( parser->pod, parser->key, neg ? "-inf" : "+inf" ) ) ) {
```

Comment:
why not just construct a inf valued float?

Reply:
@mmcgee-jump We compile with `-ffast-math`. Under that model, INF and NAN do not exist.